docs/source/user_guide/poweruser/scaling-up.rst

```diff
@@ -20,3 +20,50 @@ By default, if no address is specified Daft will spin up a Ray cluster locally o
     daft.context.set_runner_ray(address="ray://url-to-mycluster")
 
 For more information about the ``address`` keyword argument, please see the `Ray documentation on initialization <https://docs.ray.io/en/latest/ray-core/api/doc/ray.init.html>`_.
+
```

Comment:
WDYT also of renaming this document to be more explicit like "Running in distributed mode"

Reply:
I renamed it to "Distributed Computing"